src/com/owncloud/android/files/FileMenuFilter.java

```diff
@@ -59,15 +58,30 @@
      * @param cg                Accessor to app components, needed to access the
      *                          {@link FileUploader} and {@link FileDownloader} services
      * @param context           Android {@link Context}, needed to access build setup resources.
+     * @param multiSelect       Flag that indicates if a multiSelect mode is activated.
      */
     public FileMenuFilter(OCFile targetFile, Account account, ComponentsGetter cg,
-                          Context context) {
+                          Context context, boolean multiSelect) {
         mFile = targetFile;
         mAccount = account;
         mComponentsGetter = cg;
         mContext = context;
+        mMultiSelect = multiSelect;
     }
 
+    /**
+     * Constructor
+     *
+     * @param targetFile        {@link OCFile} target of the action to filter in the {@link Menu}.
+     * @param account           ownCloud {@link Account} holding targetFile.
+     * @param cg                Accessor to app components, needed to access the
+     *                          {@link FileUploader} and {@link FileDownloader} services
+     * @param context           Android {@link Context}, needed to access build setup resources.
+     */
+    public FileMenuFilter(OCFile targetFile, Account account, ComponentsGetter cg,
+                          Context context) {
+        this(targetFile, account, cg, context, false);
```

Comment:
As you can see below (in last comment), I don't see the way of canceling multiple syncing files. So, I think that if it is not supported, the cancel menu item should not be shown when selecting multiple files that are being syncing...
If anyone can check it...